pom.xml

```diff
@@ -209,7 +209,7 @@
         <dependency>
             <groupId>com.nimbusds</groupId>
             <artifactId>nimbus-jose-jwt</artifactId>
-            <version>7.9</version>
```

Comment:
Does this lib include json-smart-2.3.jar?

Reply:
Yes, we can find that out by running 
`mvn dependency:tree` command.